packages/ui/src/components/NumberInput/NumberInput.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack } from '../Stack';
+import { NumberInput, NumberInputProps } from './NumberInput';
+
+const meta: Meta = {
+  title: 'Components / NumberInput',
+  component: NumberInput,
+};
+
+export default meta;
+
+const Template: Story<NumberInputProps> = (args) => <NumberInput {...args} />;
+
+export const Playground = Template.bind({});
+
+Playground.args = {
+  size: 'sm',
+  inputMode: 'decimal',
+  disabled: false,
+};
+
+export const Sizes = () => (
+  <HStack>
+    <NumberInput size="sm" inputMode="decimal" disabled={false} />
```

Comment:
For stories showing different sizes of the NumberInput, you should focus only on the `size` prop. `inputMode` and `disabled` are no needed. This applies to other stories as well.